README.md

```diff
@@ -15,18 +15,19 @@ See [docs](https://www.fluvio.io/connectors/inbound/http/) here.
 Tutorial for [HTTP to SQL Pipeline](https://www.fluvio.io/docs/tutorials/data-pipeline/).
 
 ### Configuration
-| Option       | default                    | type    | description                                                                                |
-| :------------| :--------------------------| :-----  | :----------------------------------------------------------------------------------------- |
-| interval     | 10s                        | String  | Interval between each HTTP Request. This is in the form of "1s", "10ms", "1m", "1ns", etc. |
-| method       | GET                        | String  | GET, POST, PUT, HEAD                                                                       |
-| endpoint     | -                          | String  | HTTP URL endpoint. Use `ws://` for websocket URLs.                                         |
-| headers      | -                          | Array\<String\> | Request header(s) "Key:Value" pairs                                                |
-| body         | -                          | String  | Request body e.g. in POST                                                                  |
-| user-agent   | "fluvio/http-source 0.1.0" | String  | Request user-agent                                                                         |
-| output_type  | text                       | String  | `text` = UTF-8 String Output, `json` = UTF-8 JSON Serialized String                        |
-| output_parts | body                       | String  | `body` = body only, `full` = all status, header and body parts                             |
-| stream       | false                      | bool    | Flag to indicate HTTP streaming mode                                                       |
-| delimiter    | '\n'                       | String  | Delimiter to separate records when producing from an HTTP streaming endpoint               |
+| Option           | default                    | type            | description                                                                                |
```

Comment:
There's a lot of lines changed here to keep the table aligned, but the only modified entry is the websocket_config line.